src/zap/zapheaders.cpp

```diff
@@ -325,7 +325,7 @@ ZapPEExports::ZapPEExports(LPCWSTR dllPath)
 
 DWORD ZapPEExports::GetSize()
 {
-	return DWORD(sizeof(IMAGE_EXPORT_DIRECTORY) + wcslen(m_dllFileName) + 1);
+	return DWORD(sizeof(IMAGE_EXPORT_DIRECTORY) + wcslen(m_dllFileName) * sizeof(BYTE) + 1);
```

Comment:
It didn't like combining sizeof (byte count) and char length (element count). The scaling is to explicitly indicate that the intention is to use only one byte per char, as done in the Save() function below.